### PR TITLE
rime-sbxlm, librime-sbxlm: update license declarations

### PR DESCRIPTION
**Pull request by ZeekoZhu**

### CODE REVIEW

File: archlinuxcn/librime-sbxlm-git/PKGBUILD [hunk 1]
```diff
@@ -13,7 +13,7 @@ epoch=2
 pkgdesc="Rime input method engine (声笔系列码 GIT version)"
 arch=('x86_64')
 url="https://github.com/sbxlmdsl/librime"
-license=('GPL3')
+license=('BSD-3-Clause')
```
Comment (Zenvie) [inline archlinuxcn/librime-sbxlm-git/PKGBUILD:16]: The BSD-3-Clause license is a type of open source software license that is not part of the [standard licenses](https://gitlab.archlinux.org/archlinux/packaging/packages/licenses/-/blob/main/PKGBUILD?ref_type=heads#L40). If you use software under this license, you need to copy the original license file to the /usr/share/licenses/pkgname directory.

File: archlinuxcn/librime-sbxlm-git/PKGBUILD [hunk 2]
```diff
@@ -13,7 +13,7 @@ epoch=2
...
 depends=('boost-libs' 'capnproto' 'opencc' 'yaml-cpp' 'leveldb' 'librime-data' 'lua' 'google-glog' 'marisa')
 makedepends=('cmake' 'boost' 'git' 'gtest' 'ninja')
```
Comment (Zenvie) [inline archlinuxcn/librime-sbxlm-git/PKGBUILD:18]:
```
librime-sbxlm-git W: Dependency gcc-libs detected and implicitly satisfied (libraries ['usr/lib/libstdc++.so.6', 'usr/lib/libgcc_s.so.1'] needed in files ['usr/bin/rime_table_decompiler', 'usr/lib/rime-plugins/librime-lua.so', 'usr/bin/rime_deployer', 'usr/bin/rime_dict_manager', 'usr/lib/rime-plugins/librime-octagram.so', 'usr/lib/librime.so.1.9.0', 'usr/bin/rime_patch'])
librime-sbxlm-git W: Dependency glibc detected and implicitly satisfied (libraries ['usr/lib/libm.so.6', 'usr/lib/libc.so.6', 'usr/lib/ld-linux-x86-64.so.2'] needed in files ['usr/bin/rime_table_decompiler', 'usr/lib/rime-plugins/librime-lua.so', 'usr/bin/rime_deployer', 'usr/bin/rime_dict_manager', 'usr/lib/rime-plugins/librime-octagram.so', 'usr/lib/librime.so.1.9.0', 'usr/bin/rime_patch'])
librime-sbxlm-git W: Dependency included, but may not be needed ('capnproto')
librime-sbxlm-git W: Dependency included, but may not be needed ('librime-data')
```
ibrime-sbxlm-git-2:9.5.18.r0.g4177b219-1-x86_64-build.log
```
/build/librime-sbxlm-git/src/librime/tools/rime_api_console.cc: In function ‘int main(int, char**)’:
/build/librime-sbxlm-git/src/librime/tools/rime_api_console.cc:184:9: warning: ignoring return value of ‘int system(const char*)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
  184 |   system("chcp 65001");
      |   ~~~~~~^~~~~~~~~~~~~~
```

File: archlinuxcn/rime-sbxlm/PKGBUILD [hunk 3]
```diff
@@ -4,6 +4,7 @@ pkgrel=2
 pkgdesc='声笔系列码配置'
 arch=(any)
 source=('fetch-release.sh' 'sbxlm-init')
+license=('CC-BY-NC-4.0')
 sha256sums=('8a63e188b5f0601d881503d7d058362eda5fcfdfcf88fa6b4010c8f063c5689e'
             '496953ce12837de3f575c913a5317996002bcbf8ba2337f697b2efe15d2e0e5e')
```
Comment (Zenvie) [inline archlinuxcn/rime-sbxlm/PKGBUILD:9]:
```
38115af55a8cbe99901d9cdc187659cdcb8010d8f24cbe509a26eac24dffec85  sbxlm-init
```
```
/startdir/PKGBUILD: line 18: ./fetch-release.sh: Permission denied
curl: (2) no URL specified
curl: try 'curl --help' for more information
```

Comment (ZeekoZhu) [reply archlinuxcn/rime-sbxlm/PKGBUILD]: > /startdir/PKGBUILD: line 18: ./fetch-release.sh: Permission denied

Thanks for your review, I have double-checked the permissions on my end and can confirm that the script has the executable permission set correctly, as shown below:

```
❯ la ./fetch-release.sh 
.rwxr-xr-x 1.1Ki xxx 26 Jan  2023 ./fetch-release.sh
```

Given this, the script should execute without a permission denied error under normal circumstances. Could you please provide more details about your environment?

File: archlinuxcn/librime-sbxlm/PKGBUILD [hunk 4]
```diff
@@ -13,7 +13,7 @@ epoch=1
 pkgdesc="Rime input method engine (声笔系列码)"
 arch=('x86_64')
 url="https://github.com/sbxlmdsl/librime"
-license=('GPL3')
+license=('BSD-3-Clause')
 depends=('boost-libs' 'capnproto' 'opencc' 'yaml-cpp' 'leveldb' 'librime-data' 'lua' 'google-glog' 'marisa')
 makedepends=('cmake' 'boost' 'gtest' 'ninja')
```
Comment (Zenvie) [inline archlinuxcn/librime-sbxlm/PKGBUILD:18]:
```
librime-sbxlm W: Dependency gcc-libs detected and implicitly satisfied (libraries ['usr/lib/libgcc_s.so.1', 'usr/lib/libstdc++.so.6'] needed in files ['usr/bin/rime_patch', 'usr/lib/rime-plugins/librime-lua.so', 'usr/bin/rime_dict_manager', 'usr/lib/rime-plugins/librime-octagram.so', 'usr/bin/rime_table_decompiler', 'usr/lib/librime.so.1.9.0', 'usr/bin/rime_deployer'])
librime-sbxlm W: Dependency glibc detected and implicitly satisfied (libraries ['usr/lib/ld-linux-x86-64.so.2', 'usr/lib/libm.so.6', 'usr/lib/libc.so.6'] needed in files ['usr/bin/rime_patch', 'usr/lib/rime-plugins/librime-lua.so', 'usr/bin/rime_dict_manager', 'usr/lib/rime-plugins/librime-octagram.so', 'usr/bin/rime_table_decompiler', 'usr/lib/librime.so.1.9.0', 'usr/bin/rime_deployer'])
librime-sbxlm W: Dependency included, but may not be needed ('capnproto')
librime-sbxlm W: Dependency included, but may not be needed ('librime-data')
```

File: archlinuxcn/librime-sbxlm/PKGBUILD [hunk 5]
```diff
@@ -47,4 +47,5 @@ check() {
 package() {
   cd "${_pkgname}-${pkgver}/build"
```
Comment (Zenvie) [inline archlinuxcn/librime-sbxlm/PKGBUILD:48]:
```
[85/156] Building CXX object tools/CMakeFiles/rime_api_console.dir/rime_api_console.cc.o
/build/librime-sbxlm/src/librime-9.5.18/tools/rime_api_console.cc: In function ‘int main(int, char**)’:
/build/librime-sbxlm/src/librime-9.5.18/tools/rime_api_console.cc:184:9: warning: ignoring return value of ‘int system(const char*)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
  184 |   system("chcp 65001");
      |   ~~~~~~^~~~~~~~~~~~~~
```

File: archlinuxcn/rime-sbxlm/PKGBUILD (hunk 3 above)
Comment (Zenvie) [reply archlinuxcn/rime-sbxlm/PKGBUILD]: The environment is a build machine provided by archlinuxcn, and I use the command extra-x86_64-build to compile. Maybe the build machine has some restrictions?

@lilydjwg Sorry to bother you, but can you please take a look and help me solve this?

**Comment by Zenvie** [comment]
@lilydjwg I apologize for the inconvenience caused by my mistake. I accidentally terminated the program, which resulted in file locking. I also mistakenly entered the -r parameter when typing the command, so I might need to trouble you to delete the folder in my home directory (I know this is the second time, ~~and I won't do it again~~). In addition, could you please help me change my username on the compiler machine? Thank you for your understanding and cooperation.

**Comment by lilydjwg** [comment]
Done. New username is `zenvie`.

File: archlinuxcn/rime-sbxlm/PKGBUILD (hunk 3 above)
Comment (lilydjwg) [reply archlinuxcn/rime-sbxlm/PKGBUILD]: Where did you see this error? Do you have a complete log?

Comment (Zenvie) [reply archlinuxcn/rime-sbxlm/PKGBUILD]: > Where did you see this error? Do you have a complete log?

```
build% extra-x86_64-build
:: Synchronizing package databases...
 core downloading...
 extra downloading...
:: Starting full system upgrade...
 there is nothing to do
==> Building in chroot for [extra] (x86_64)...
==> Synchronizing chroot copy [/var/lib/archbuild/extra-x86_64/root] -> [zenvie]...done
==> Making package: rime-sbxlm 20231216-2 (Tue Feb  6 08:49:39 2024)
==> Retrieving sources...
  -> Found fetch-release.sh
  -> Found sbxlm-init
==> Validating source files with sha256sums...
    fetch-release.sh ... Passed
    sbxlm-init ... Passed
==> Making package: rime-sbxlm 20231216-2 (Tue Feb  6 08:49:41 2024)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Installing missing dependencies...
resolving dependencies...
looking for conflicting packages...

Package (3)      New Version  Net Change

extra/oniguruma  6.9.9-1        0.87 MiB
extra/jq         1.7.1-1        0.74 MiB
extra/unzip      6.0-20         0.30 MiB

Total Installed Size:  1.90 MiB

:: Proceed with installation? [Y/n]
checking keyring...
checking package integrity...
loading package files...
checking for file conflicts...
:: Processing package changes...
installing oniguruma...
installing jq...
installing unzip...
==> Retrieving sources...
  -> Found fetch-release.sh
  -> Found sbxlm-init
==> WARNING: Skipping all source file integrity checks.
==> Extracting sources...
==> Starting prepare()...
/startdir/PKGBUILD: line 18: ./fetch-release.sh: Permission denied
curl: (2) no URL specified
curl: try 'curl --help' for more information
==> ERROR: A failure occurred in prepare().
    Aborting...
==> ERROR: Build failed, check /var/lib/archbuild/extra-x86_64/zenvie/build
```

Comment (lilydjwg) [reply archlinuxcn/rime-sbxlm/PKGBUILD]: Weird. Can you check the file permission?

File: archlinuxcn/rime-sbxlm/PKGBUILD [hunk 6]
```diff
@@ -4,8 +4,9 @@ pkgrel=2
 pkgdesc='声笔系列码配置'
 arch=(any)
 source=('fetch-release.sh' 'sbxlm-init')
+license=('CC-BY-NC-4.0')
 sha256sums=('8a63e188b5f0601d881503d7d058362eda5fcfdfcf88fa6b4010c8f063c5689e'
-            '496953ce12837de3f575c913a5317996002bcbf8ba2337f697b2efe15d2e0e5e')
+            '38115af55a8cbe99901d9cdc187659cdcb8010d8f24cbe509a26eac24dffec85')
 makedepends=('jq' 'unzip')
 optdepends=('librime-sbxlm-git' 'fcitx5-rime')
```
Comment (Zenvie) [inline archlinuxcn/rime-sbxlm/PKGBUILD:11]:
```
rime-sbxlm E: Dependency bash detected and not included (programs ['bash'] needed in scripts ['usr/bin/sbxlm-init'])
```

File: archlinuxcn/rime-sbxlm/PKGBUILD [hunk 7]
```diff
@@ -4,8 +4,9 @@ pkgrel=2
 pkgdesc='声笔系列码配置'
 arch=(any)
 source=('fetch-release.sh' 'sbxlm-init')
```
Comment (Zenvie) [inline archlinuxcn/rime-sbxlm/PKGBUILD:6]:
```
PKGBUILD (rime-sbxlm) W: Missing Maintainer tag
PKGBUILD (rime-sbxlm) E: Missing url
```

File: archlinuxcn/rime-sbxlm/PKGBUILD (hunk 3 above)
Comment (Zenvie) [reply archlinuxcn/rime-sbxlm/PKGBUILD]: > Weird. Can you check the file permission?

Sorry for the trouble, it turns out that the file permissions are insufficient.

**Pull request merged**